tests/test_SAME.py

```diff
@@ -424,3 +425,13 @@ def test_reconcile_character(self):
         self.assertEqual((2, 'L'), SAME._reconcile_character(bitstrue, bitsfalse,
                                                              'ABCDEFGHIJKLMNOPQRSTUVWXYZ'))
 
+    def test_mutate_string(self):
```

Comment:
I was referring to the now absent mutate_string(), but since it's absent, no reason to worry about that duplication!  (Do you want to eliminate mutate_string in both places?)